packages/app/src/breadcrumbs/utils.ts

```diff
@@ -0,0 +1,21 @@
+import { version } from '../../package.json';
+
+export function prepareFeedback(filepath: string, entityPath: string): string {
+  return `Hi,
+
+<< Please provide your feedback here. >>
+<< To report an issue, please include screenshots, reproduction steps, etc. >>
+<< To suggest a new feature, please describe the needs this feature would fulfill. >>
```

Comment:
I would add a line saying `You can also open an issue on https://github.com/silx-kit/h5web/issues` to make use of the issue template.

Speaking of which, I would also be in favour of adding a feature request template like for jupyterlab-h5web: https://github.com/silx-kit/jupyterlab-h5web/blob/main/.github/ISSUE_TEMPLATE/feature_request.md

Reply:
I was going to add the line, but I'm having second thoughts. I'm wondering whether this wouldn't just confuse users. They're about to write an email and then we tell them "oh, actually, maybe just create an issue on GitHub" -- they may end up trying to create an issue, look at the issue template and give up; they may have trouble remembering their GitHub credentials are giving up at the 2FA step; worse, they may open the issue but forget to paste the contextual information pre-filled in the email body!

FYI, I've just discovered that it's possible to generate GitHub URLs to create issues with pre-filled content: https://github.com/silx-kit/h5web/issues/new?title=Foo&body=Bar. You can even select a template: https://github.com/silx-kit/h5web/issues/new?title=Foo&template=bug_report.md. Unfortunately, you can't do both - i.e. pre-fill a template with contextual information; otherwise I would have been happy to add a link with such a URL in the feedback email.

In conclusion, I think the pre-filled email with no mention of GitHub is the least confusing solution that maximises our chances of receiving feedback.

Reply:
> In conclusion, I think the pre-filled email with no mention of GitHub is the least confusing solution that maximises our chances of receiving feedback.

Sounds good